Dockerfile

```diff
@@ -48,15 +48,6 @@ RUN pip install -r requirements.txt
 RUN dnf remove ${BUILD_PACKAGES} -y && \
     dnf clean all
 
-# Fetch UI code
-# TODO FIXME REMOVE THE UI CODE!
-# This bundles an old version of the UI while we are release a new version separately.
-# After we have released the "new" UI code and proven it to be stable, we must remove
```

Comment:
I'm just leaving this comment here as an unresolved thread so we don't accidentally merge this _before_ we have finished shipping 1.11. 😉